src/tls13.c

```diff
@@ -11999,7 +12017,31 @@ int Tls13UpdateKeys(WOLFSSL* ssl)
         return 0;
 #endif /* WOLFSSL_DTLS13 */
 
-    return SendTls13KeyUpdate(ssl);
+    return SendTls13KeyUpdate(ssl, resp);
+}
+
+/* Update the keys for encryption.
+ * If using non-blocking I/O and WOLFSSL_ERROR_WANT_WRITE is returned then
+ * calling wolfSSL_write() will have the message sent when ready.
+ *
+ * ssl  The SSL/TLS object.
+ * returns BAD_FUNC_ARG when ssl is NULL, or not using TLS v1.3,
+ * WOLFSSL_ERROR_WANT_WRITE when non-blocking I/O is not ready to write,
+ * WOLFSSL_SUCCESS on success and otherwise failure.
+ */
+int wolfSSL_update_enc_keys(WOLFSSL* ssl)
+{
+    int ret;
+
+    if (ssl == NULL || !IsAtLeastTLSv1_3(ssl->version))
+        return BAD_FUNC_ARG;
+
+    ret = Tls13UpdateKeys(ssl, 0);
```

Comment:
I ran the suggested test case and did NOT see `SendTls13KeyUpdate` logged with debug enabled. Should I see a key update? Do I need to call `wolfSSL_CTX_set_auto_rekey`?